homeassistant/helpers/config_validation.py

```diff
@@ -402,3 +402,9 @@ def validator(value):
     [vol.Any(SERVICE_SCHEMA, _SCRIPT_DELAY_SCHEMA, EVENT_SCHEMA,
              CONDITION_SCHEMA)],
 )
+
+
+def string_schema(*args: Sequence) -> Dict:
+    """Return a schema dict of Required values or Optional values."""
+    return {val if isinstance(val, vol.Optional) else vol.Required(val): string
+            for val in args}
```

Comment:
I have no idea about the percentage of a validation for required strings. The guess is 15 to 20 %. The downside seems that this would only be useful for "required strings" as for optional ones there is in a lot cases a default value involved.